src/components/SearchResult.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+const SearchResult = props => (
+  <div className="search">
+    <div className="page-header">
+      <h4 className="text-left">Results</h4>
+    </div>
+    <div className="row">
+      <div className="col">
+        <table className="table search-table">
+          <thead>
+            <tr>
+              <th>Title</th>
+              <th>First name</th>
+              <th>Sur name</th>
+              <th>Email</th>
+              <th />
+              <th>Room id</th>
+              <th>Check in date</th>
+              <th>Check out date</th>
+            
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
```

Comment:
Can you see an opportunity for extracting a component in the future here? Is there a repeating pattern?